infra/modules/github_selfhosted_runner_on_container_app_jobs/locals.tf

```diff
@@ -0,0 +1,11 @@
+locals {
```

Comment:
Now that I understand better what you meant, I'm skeptical about hardcoding project specific values into shared modules (think about the `dx` repos as an open source general purpose project). AFAIK this are the only project specific settings so far.

Reply:
I restored previous config. I wonder if also the secret name should be hardcoded or not.
My proposal is to create an optional variable with a default value

Reply:
I agree.